pages/index.js

```diff
@@ -12,21 +12,29 @@ export default function HomePage() {
   }
 
   return (
-    <StyledList>
-      {isLoading ? (
-        <li>
-          <h2>Loading...</h2>
-        </li>
-      ) : (
-        items.map((item) => (
-          <li key={item.itemId}>
-            <StyledLink href={`/items/${item.itemId}`}>
-              <Item title={item.title} initialStatus={item.initiallyLost} />
-            </StyledLink>
+    <>
+      <AddItemLink href={"/create"}>
```

Comment:
this link should work fine without the brackets

```suggestion
      <AddItemLink href="/create">
```